src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/controllers/api/QueueController.kt

```diff
@@ -18,7 +18,7 @@ class QueueController(
             this(
                     context,
                     AppConfig(),
-                    OrderlyServerClient(AppConfig()).throwOnError()
+                    OrderlyServerClient(AppConfig())
```

Comment:
..same for removing throwOnError here?

Reply:
Yeah, unclear to me why we ever had this here.